lib/genesis/genesis.go

```diff
@@ -22,29 +22,31 @@ import (
 
 // Genesis stores the data parsed from the genesis configuration file
 type Genesis struct {
-	Name            string                 `json:"name"`
-	ID              string                 `json:"id"`
-	ChainType       string                 `json:"chainType"`
-	Bootnodes       []string               `json:"bootNodes"`
-	ProtocolID      string                 `json:"protocolId"`
-	Genesis         Fields                 `json:"genesis"`
-	Properties      map[string]interface{} `json:"properties"`
-	ForkBlocks      []string               `json:"forkBlocks"`
-	BadBlocks       []string               `json:"badBlocks"`
-	ConsensusEngine string                 `json:"consensusEngine"`
+	Name               string                 `json:"name"`
+	ID                 string                 `json:"id"`
+	ChainType          string                 `json:"chainType"`
+	Bootnodes          []string               `json:"bootNodes"`
+	TelemetryEndpoints []interface{}          `json:"telemetryEndpoints"`
```

Comment:
I'd rather not use `[]interface{}`, I'd rather make a struct like 
```
type TelemetryEndpoint struct {
	Endpoint string
	Level    float64
}
```
which I tried, but this caused an error when with `json.Unmarshal`  when decoding (from polkadot genesis-raw.json): 
```
"telemetryEndpoints": [
    [
      "wss://telemetry.polkadot.io/submit/",
      0
    ]
  ],
```
Any suggestions on how I can better handle this?

Reply:
it seems like the endpoints are tuples/arrays (use `[ ]` instead of `{ }`) and I think Go expects `[ ]` to be an `[]interface{}` instead of a `struct`, so not sure if there's a better way to do this..

Reply:
Right, it's working, so I guess this is the way to go.